### ISSUE TITLE
Does picom work with the non `code-oss` VS Code flavours?

### ISSUE THREAD

**Issue by wthrajat** [yshui/compton#1161]
```sh
opacity-rule = [
  "100:class_g    = 'Alacritty'",
  "100:class_g     = 'Polybar'",
  "85:class_g    = 'code-oss'",
  "100:class_g    = 'Meld'",
  "100:class_g    = 'firefox'",
  "100:class_g    = 'Thunderbird'"
];
```
(Please have a look at https://wiki.archlinux.org/title/Visual_Studio_Code for complete context)

Earlier when I was using the `code-oss` version, it used to work well (reduced capacity and great blur) because I'd put `code-oss` in the above rule.
Now I have switched to the official VSCode binary (`visual-studio-code-bin` in AUR)
  
Now I am not sure what should I put there instead of `code-oss`, I have tried to put `visual-studio-code-bin` but it doesn't work then.

Tried googling but everyone seems to use `code-oss` and I cannot use `code-oss` now for some reason, hence was wondering if I could use picom with the official binary?

TLDR; is there a way to use picom with the non `code-oss` flavour of VS Code?
Thanks a lot! : )

**Comment by yshui** [comment]
You can use `xprop` to find out the class of a window.

**Comment by wthrajat** [comment]
That worked, thanks a lot!
Also great work `picom`!
